helm/aio-connector-multidataset/sample-datasets.yaml

```diff
@@ -1,25 +1,33 @@
 datasets:
   - name: thermodynamics
-    datasetConfiguration: |-
```

Comment:
We need to bring this back i.e. dataset level config for polling; my bad, in my last example I must have used the datapoint level config.

Reply:
Another PR?